packages/wattpm/lib/commands/build.js

```diff
@@ -86,6 +86,56 @@ export async function installDependencies (logger, root, services, production, p
   }
 }
 
+async function updateDependencies (logger, availableVersions, path, target) {
+  // Parse the configuration file, if any
+  const packageJsonPath = resolve(path, 'package.json')
+
+  if (!existsSync(packageJsonPath)) {
+    return
+  }
+
+  let updated = false
+  const packageJson = JSON.parse(await readFile(packageJsonPath, 'utf-8'))
+
+  for (const section of ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies']) {
+    const sectionLabel = section === 'dependencies' ? '' : ` (${bold(section)})`
+    for (const [pkg, range] of Object.entries(packageJson[section] ?? {})) {
+      const specifier = range[0]
+
+      if (!packages.includes(pkg)) {
+        continue
+      }
+
+      if (specifier !== '^' && specifier !== '~') {
+        continue
```

Comment:
I think we should throw an error instead because we couldn't complete the operation and provide a `--force` to always change this. Most of the times I wanted to massively update those are when they were fixed versions.

Reply:
I see. Will add this!

Reply:
Added!